test/RdfParserTest.cpp

```diff
@@ -861,6 +874,17 @@ auto forAllParsers(const auto& function, const auto&... args) {
   forAllMultifileParsers(function, args...);
 }
 
+struct TurtleStreamAndParallelWithParser {
```

Comment:
Here we should also aim to minimize the diff by rewriting the 
`forAllParsers` etc. lambdas to get the parser as a `type_identity` ,s t. we can minimize the diff that is to be reviewed.

Reply:
Addressed